.github/utils/update_devcontainer.py

```diff
@@ -0,0 +1,46 @@
+import os
+import argparse
+
+ECR_REPO = "337532070941.dkr.ecr.us-east-1.amazonaws.com"
+DOCKER_PREFIX = "ugbio"
+
+def update_image_in_file(file_path, workspace, new_version):
+    new_image = f"{ECR_REPO}/{DOCKER_PREFIX}_{workspace}:{new_version}"
```

Comment:
I think you can use json parsing instead. Find the "image" entry and replace it with just the version (whatever comes after ":"). This way we are agnostic to the ecr repo and the folder containing the devconatriner.json can have whatever name it wants. 

Note that one devconatiner.json doesn't contain "image" so you should skip it

Reply:
done, added a prior check.
updating using json package is not an option - because it doesn't preserve the comments in the json